devicetypes/smartthings/zwave-button.src/zwave-button.groovy

```diff
@@ -0,0 +1,148 @@
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition (name: "Z-Wave Button", namespace: "smartthings", author: "SmartThings") {
+		capability "Button"
+		capability "Battery"
+		capability "Sensor"
+		capability "Health Check"
+		capability "Configuration"
+
+		fingerprint mfr: "010F", prod: "0F01", model: "1000", deviceJoinName: "Fibaro Button"
+		fingerprint mfr: "0086", prod: "0001", model: "0026", deviceJoinName: "Aeotec Panic Button"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "button", type: "generic", width: 6, height: 4, canChangeIcon: true) {
+			tileAttribute("device.button", key: "PRIMARY_CONTROL") {
+				attributeState "default", label: ' ', icon: "st.unknown.zwave.remote-controller", backgroundColor: "#ffffff"
+			}
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label:'${currentValue}% battery', unit:""
+		}
+
+		main "button"
+		details(["button", "battery"])
+	}
+}
+
+def installed() {
+	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	sendEvent(name: "numberOfButtons", value: 1)
+	response([
+			secure(zwave.batteryV1.batteryGet()),
+			"delay 2000",
+			secure(zwave.wakeUpV1.wakeUpNoMoreInformation())
+	])
+}
+
+def configure() {
+	if(zwaveInfo.mfr?.contains("0086"))
+		[
+			secure(zwave.configurationV1.configurationSet(parameterNumber: 250, scaledConfigurationValue: 1)),
+			secure(zwave.associationV1.associationSet(groupingIdentifier: 1, nodeId:zwaveHubNodeId))
+		]
+}
+
+def parse(String description) {
+	def result = []
+	if (description.startsWith("Err")) {
+		result = createEvent(descriptionText:description, isStateChange:true)
+	} else {
+		def cmd = zwave.parse(description, commandClasses)
+		if (cmd) {
+			result += zwaveEvent(cmd)
+		}
+	}
+	log.debug "Parse returned: ${result}"
+	result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand(commandClasses)
+	if (encapsulatedCommand) {
+		zwaveEvent(encapsulatedCommand)
+	} else {
+		log.warn "Unable to extract encapsulated cmd from $cmd"
+		createEvent(descriptionText: cmd.toString())
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.centralscenev1.CentralSceneNotification cmd) {
+	def value = eventsMap[(int) cmd.keyAttributes]
+	def event = createEvent(name: "button", value: value, descriptionText: "Button was ${value}")
```

Comment:
Please include `data: [buttonNumber: 1]` in all the button events you send. I'm not quite sure if it's necessary, but we're including it for now.